environment/src/main/java/com/sequenceiq/environment/network/NetworkService.java

```diff
@@ -75,8 +75,8 @@ private CloudPlatform getCloudPlatform(Environment environment) {
 
     public BaseNetwork validate(BaseNetwork originalNetwork, EnvironmentEditDto editDto, Environment environment) {
         if (originalNetwork.getRegistrationType() == RegistrationType.CREATE_NEW) {
-            throw new BadRequestException("Subnet could not be attached to this environment, because it is newly created by Cloudbreak. " +
-                    "You need to re-install the the environment into an existing VPC");
+            throw new BadRequestException("Subnet could not be attached to this environment, because it has been created by Cloudera. " +
+                    "You need to re-install the environment into an existing VPC/VNet");
```

Comment:
official link for the documentation should be fine.

Reply:
This was just a wording fix :D 
Anyways, I have a hard time finding this in the docs, have you been thinking about this one? https://docs.cloudera.com/management-console/cloud/environments/topics/mc-subnet-adding-azure.html